virtual_rainforest/models/litter/litter_pools.py

```diff
@@ -0,0 +1,162 @@
+"""The ``models.litter.litter_pools`` module  simulates the litter pools for the Virtual
+Rainforest. At the moment only two pools are modelled, above ground metabolic and above
+ground structural, but a wider variety of pools will be simulated in future.
+"""  # noqa: D205, D415
+
+import numpy as np
+from numpy.typing import NDArray
+from xarray import DataArray
+
+from virtual_rainforest.models.litter.constants import LitterConsts
+
+# TODO - At the moment this module does not use litter chemistry (relative lignin
+# content) at all. We need to decide how we handle this and adjust the below functions
+# to use this at some point.
+
+
+def calculate_litter_pool_updates(
+    surface_temp: NDArray[np.float32],
+    litter_pool_above_metabolic: NDArray[np.float32],
+    litter_pool_above_structural: NDArray[np.float32],
+    update_interval: float,
+    constants: LitterConsts,
+) -> dict[str, DataArray]:
+    """Calculate updates for all litter pools.
+
+    Args:
+        surface_temp: Temperature of soil surface, which is assumed to be the same
+            temperature of the above ground litter [C]
+        litter_pool_above_metabolic: Above ground metabolic litter pool [kg C m^-2]
+        litter_pool_above_structural: Above ground structural litter pool [kg C m^-2]
+        update_interval: Interval that the litter pools are being updated for [days]
```

Comment:
I remember being involved in some discussion about this a while back... but did you guys arrive at an agreement about using a common time unit throughout?

Reply:
I don't know if we ever reached an agreement on common time units, we agreed that all models would use the same time step for updates (i.e. that no model should update the data object faster than the other models). I don't think we've settled on common units for anything, though we probably need to as part of sorting #129

Reply:
That's fine. Just wanted to check. You've been explicit about what your time units are, which is all you really can do in this case.